module/entityAssessors/helper.js

```diff
@@ -802,18 +806,120 @@ module.exports = class EntityAssessorHelper {
         })
     }
 
+      /**
+     * Create entity assessor data.
+     * @method
+     * @name create
+     * @param {String} programId - program id.
+     * @param {String} solutionId - solution id.
+     * @param {Array}  entities - entities data.  
+     * @returns {Object} create entity assessor data.
+     */
+
+    static create(
+        userDetails,
+        programId,
+        solutionId,
+        entities = []
+    ) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let solutionData = 
+                await solutionsHelper.solutionDocuments({ 
+                    _id : solutionId,
+                    status : messageConstants.common.ACTIVE_STATUS
+                },["_id","entityType","entityTypeId"]);
+
+                if( !solutionData.length > 0) {
+                    throw {
+                        status : httpStatusCode.bad_request.status,
+                        message : messageConstants.apiResponses.SOLUTION_NOT_FOUND
+                    }
+                }
+
+                let programData = await programsHelper.list({
+                    _id : programId
+                },["_id"]);
+
+                if ( !programData.length > 0 ) {
+                    throw {
+                        status : httpStatusCode.bad_request.status,
+                        message : messageConstants.apiResponses.PROGRAM_NOT_FOUND
+                    }
+                }
+
+                let assessorData = await this.assessorsDocument({
```

Comment:
@aman-tunerlabs - If this is create only why check if the existing document is there or not?